lib/util/cache/repository/impl/client/local/index.spec.ts

```diff
@@ -20,25 +21,24 @@ async function createCacheRecord(
   const hash = hasha(jsonStr, { algorithm: 'sha256' });
   const compressed = await compress(jsonStr);
   const payload = compressed.toString('base64');
-  const record: RepoCacheRecord = { revision, repository, payload, hash };
-  return record;
+  return { revision, repository, payload, hash };
 }
 
-describe('util/cache/repository/impl/local', () => {
+describe('util/cache/repository/impl/client/local/index', () => {
```

Comment:
why we need the client sub folder?

Reply:
to group all client implementations and differentiate them from the other factory and cache impl. 
can be removed if desired, its just a preference

Reply:
i don't think we'll have any other types here in future, so remove.

Reply:
are you referring to the `local` folder i.e. rename `index.ts` -> `local.ts`.
in this case we do need the `client` folder